components/resources/rewards_strings.grdp

```diff
@@ -526,5 +537,17 @@
   <message name="IDS_REWARDS_TIP_DEFAULT_CREATOR_DESCRIPTION" desc="">
     Welcome! Users can support this content creator by sending them contributions. It’s a way of thanking them for making great content.
   </message>
+  <message name="IDS_REWARDS_TIP_SELF_CUSTODY_TITLE" desc="">
+    Support your favorite creators
+  </message>
+  <message name="IDS_REWARDS_TIP_SELF_CUSTODY_HEADER" desc="">
+    Show your love and send a token of your gratitude
+  </message>
+  <message name="IDS_REWARDS_TIP_SELF_CUSTODY_TEXT" desc="">
+    Use your web3 wallet to send a one time contribution to this creator and show them your appreciation
```

Comment:
Nit: See other comment on Web3 capitalization.

Reply:
We should capitalize Web3 in all cases, no matter its placement in a sentence.